WES+WGS/Pipeline.nf

```diff
@@ -30,16 +29,16 @@ process HarmonizeSM {
 
 // Merge the WES and WGS files
 process Merge {
-   cache "lenient"
+   cache 'lenient'
+   
    cpus 1
-   memory "16 GB"
-   time "6h"
+   memory "1 GB"   
+   time '2h'
    errorStrategy 'retry'
-   maxRetries 1
-   
+   maxRetries 3
```

Comment:
Doesn't make sense to have maxRetries 3.... Values apply only to your situation

Reply:
reverted